Extension/CHANGELOG.md

```diff
@@ -1,35 +1,24 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.29.0-insiders2: July 8, 2020
+## Version 0.29.0: July 15, 2020
 ### New Features
-* Added support for LogMessage Breakpoints for debug type `cppdbg`. [#1013](https://github.com/microsoft/MIEngine/pull/1013)
-* Switch to using the VS Code Semantic Tokens API for semantic colorization. [PR #5401](https://github.com/microsoft/vscode-cpptools/pull/5401)
+* Add Doxygen comment support (to tooltip display of hover, completion, and signature help). [#658](https://github.com/microsoft/vscode-cpptools/issues/658)
+  * Add `C_Cpp.simplifyStructuredComments` setting. [#5706](https://github.com/microsoft/vscode-cpptools/issues/5706)
```

Comment:
The link to the issue on this sub bullet is not needed because it's part of the parent feature.
You can just mention "The way comments are formatted is controlled by the `C_Cpp.simplifyStructuredComments` setting."